sesame/src/main/scala/org/w3/banana/sesame/SesameTransactor.scala

```diff
@@ -0,0 +1,28 @@
+package org.w3.banana.sesame
+
+import org.openrdf.repository.RepositoryConnection
+import org.w3.banana.{Transactor, SparqlUpdate, RDFStore}
+
+import scala.util.{Failure, Try}
+
+trait SesameTransactor extends Transactor[Sesame,RepositoryConnection] {
+
+  def r[T](conn: RepositoryConnection, body: => T): Try[T] = Try{
+    val result = body
+    conn.close()
```

Comment:
I had I discussion with one of Bigdata founders and he explained to me that connection opening/closing is very cheap there and that he recommends to close the connection for each transaction I make cause many open connections lead to trouble in bigdata.